content/authors/adkris/_index.md

```diff
@@ -66,11 +65,11 @@ user_groups:
 - Researchers
 ---
 
-I am a student at the University of Washington majoring in Electrical Engineering with a concentration in Digital Signal Processing. I am expected to graduate the Spring of 2024 and am interested in pursuing graduate research after my bachelor's degree.
+I am a student at the University of Washington majoring in Electrical Engineering and pursuing Digital Signal Processing and Neural Engineering concentrations. I am expected to graduate the Spring of 2024 and am interested in pursuing graduate research after my bachelor's degree.
 
-My core competency is in programming, with an expertise in Java and Python. I aim to use my programming skills to aid in ecological & environmental conservation. 
+My core competency is in programming, with an expertise in Python and Java. I am currently studying how signal processing methods are used in passive acoustic monitoring projects to address project-specific goals and aid in ecological conservation.
 
-At Echospace, I am currently working on using machine learning methods on echolocation data to understand the foraging and social behaviors of bats in the wild. 
+At Echospace, I am developing and applying computational methods on echolocation data to understand the foraging and social behaviors of bats in the wild. We collect our echolocation data from local bat colonies by deploying AudioMoths, which are autonomous recording units, at the Union Bay Natural Area, beside the Husky Football Stadium.
```

Comment:
```suggestion
At Echospace, I am developing and applying computational methods on echolocation data to understand the foraging and social behaviors of bats in the wild. We collect our echolocation data from local bat colonies by deploying autonomous recording units (AudioMoths [ADD LINK]), at the Union Bay Natural Area [ADD LINK] right bu the UW Husky Football Stadium.
```
The ADD LINK parts are suggestions, you can decide if you actually want them hyperlinked.

Reply:
Added a link to the Audiomoth page! Good suggestion!